src/components/utils/test/date_time_test.cc

```diff
@@ -108,11 +108,10 @@ TEST(DateTimeTest, CalculateTimeSpan) {
   // arrange
   const TimevalStruct time = date_time::DateTime::getCurrentTime();
 
-  const uint32_t sleep_time_mSec = 10;
+  const uint64_t sleep_time_mSec = 100;
```

Comment:
@OHerasym I also don't like idea ti increase timeout due to unknown reason of QT behaviour